crates/sui-move/src/build.rs

```diff
@@ -28,9 +28,9 @@ pub struct Build {
     /// and events.
     #[clap(long, global = true)]
     pub generate_struct_layouts: bool,
-    /// If `true`, enable linters
+    /// If `true`, disable linters
     #[clap(long, global = true)]
-    pub lint: bool,
+    pub no_lint: bool,
```

Comment:
Could you add a short test with this enabled? It want to make sure the name is `--no-lint` and not `--no_lint`. Discovered this in the Move CLI codebase and am now very paranoid :)